README.md

```diff
@@ -29,6 +29,40 @@ Tutorials in the **Intro-Full** directory cover
 
 All the tutorial folders can be built using either the `Makefile` or the CMake `CMakeLists.txt` file in each folder.
 
+## CMake
+
+CMake can build against an installed Kokkos library or download one automatically using `FetchContent`.
+
+Without any Kokkos already installed in an exercise directory, one can run the following:
+
+```shell
+cmake -B build_dir . # -DKokkos_* options
```

Comment:
```suggestion
cmake -B build_dir # -DKokkos_* options
```
The path to the current working directory is implicit.  Otherwise we probably should teach using `-S <source_dir>`